code/index.html

```diff
@@ -3,28 +3,96 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css">
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+       <!-- 1. Hero section -->
+   <header class="hero">
+     <picture>
+       <source srcset="./Images/Img.mobile.jpg" media="(max-width: 767px)">
+       <source srcset="./Images/Img.tablet.jpg" media="(max-width: 1199px)">
+       <img class="hero-image" src="./Images/Img.desktop.jpg" alt="Hero image" />
+      </picture>
+      <h1 class="hero-title">Sunshine Travels</h1>
+   </header> 
 
+   <section class="info-section">
     <!-- Signup form -->
+    <form class="form" action="https://httpbin.org/anything" method="POST">
+      
+      <h2>Sign up/ log in</h2>
+      <label>
+        Name: <input name= "form-name" type="text" required>
+      </label>
+      <br></br>
```

Comment:
No need for the space after the equal sign. Try to omit all `<br>`s as they are not semantic. You can style the `label` with some margin to achieve the same thing